### ISSUE TITLE
Cython version too old

### ISSUE THREAD

**Issue by katyhuff** [cyclus/cyclus#1374]
- I have a fresh, clean ubuntu 16.04 virtual machine. 
- I have followed install from source instructions exactly, installing everything, including all optional packages, with apt-get. That includes `cython3`, as according to the instructions.
- I created a directory called cyclus. In it, created a directory called install.
- In the cyclus directory, I cloned cyclus and cycamore. 
- From the cyclus/cyclus directory, I executed python install.py --prefix=../install

Finally, I received:

```
-- Cython version 0.23.4
Cmake Error at CMakeLists.txt:279 (message):
  Cython version is too old, must be 0.25+
```

So, the debian stable version of cython is .23.4 but we require .25+, so the installation instructions probably need to reflect that you have to pip update cython thus: http://cython.readthedocs.io/en/latest/src/quickstart/install.html

**Comment by scopatz** [comment]
Or conda install cython

**Comment by katyhuff** [comment]
Yes. correct. Right now, the dependency installation instructions seem to ignore conda... http://fuelcycle.org/user/DEPENDENCIES.html

**Comment by katyhuff** [comment]
The cython treatment will need to be changed in the docs here:
https://github.com/cyclus/cyclus/blob/develop/DEPENDENCIES.rst

@skc222 is on it.

**Comment by scopatz** [comment]
Cool thanks @katyhuff and @skc222!

**Comment by katyhuff** [comment]
This has been closed with #1378 .
